packages/evm/src/opcodes/codes.ts

```diff
@@ -210,6 +210,14 @@ const opcodes: OpcodeEntry = {
   0xa3: dynamicGasOp('LOG'),
   0xa4: dynamicGasOp('LOG'),
 
+  // '0xf0' range - extended range/width modular arithmetic
+  0xc0: asyncAndDynamicGasOp('SETMODX'),
```

Comment:
This will add these opcodes as available from all EVMs at all hardforks (so these opcodes are now available at Frontier / Chainstart)